linode/instancereservedipassignment/framework_models.go

```diff
@@ -0,0 +1,108 @@
+package instancereservedipassignment
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/linodego"
+
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+	"github.com/linode/terraform-provider-linode/v2/linode/instancenetworking"
+)
+
+type InstanceIPModel struct {
+	ID               types.String `tfsdk:"id"`
+	LinodeID         types.Int64  `tfsdk:"linode_id"`
+	Public           types.Bool   `tfsdk:"public"`
+	Address          types.String `tfsdk:"address"`
+	Gateway          types.String `tfsdk:"gateway"`
+	Prefix           types.Int64  `tfsdk:"prefix"`
+	RDNS             types.String `tfsdk:"rdns"`
+	Region           types.String `tfsdk:"region"`
+	SubnetMask       types.String `tfsdk:"subnet_mask"`
+	Type             types.String `tfsdk:"type"`
+	ApplyImmediately types.Bool   `tfsdk:"apply_immediately"`
+	IPVPCNAT1To1     types.List   `tfsdk:"vpc_nat_1_1"`
+	Reserved         types.Bool   `tfsdk:"reserved"`
+}
+
+func (m *InstanceIPModel) FlattenInstanceIP(
```

Comment:
```suggestion
func (m *InstanceIPModel) flattenInstanceIP(
```
This function looks like only used within this package.

Reply:
You'll have to change them in the usage as well :)